### PR TITLE
Track signup events

### PR DESCRIPTION
**Pull request by robbiecarlton**
Goes with https://github.com/Hylozoic/hylo-redux/pull/92

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - good to merge these two?

File: api/services/Analytics.js [hunk 1]
```diff
@@ -31,4 +31,14 @@ instance.pixelUrl = function (emailName, props) {
   return prefix + encodedData
 }
 
+instance.trackSignup = function (userId, req) {
+  let event = 'Signup success'
+  if (req.headers['ios-version']) {
+    event += ' (iOS)'
```
Comment (levity) [inline api/services/Analytics.js]: Don't change the event name; change the value of a property instead.

Comment (robbiecarlton) [inline api/services/Analytics.js]: done

**Comment by levity** [comment]
The important thing to test is that Mixpanel treats client- and server-side events as belonging to the same user. I think this is set up due to the way we make the `identify` call, but it may need to be repeated or done specifically after signup. The way to test this would be to look in Mixpanel at the stream of events for a test user, and confirm that an event before signup, the signup event, and an event after signup are all correctly assigned to that user.

**Comment by levity** [comment]
I'm going to merge this locally and test what I described above

**Pull request merged**